samples/hello-kapt/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+import org.gradle.api.internal.HasConvention
+import org.gradle.kotlin.dsl.delegateClosureOf
+import org.jetbrains.kotlin.gradle.plugin.KaptAnnotationProcessorOptions
+import org.jetbrains.kotlin.gradle.plugin.KaptJavacOptionsDelegate
+import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
+
+plugins {
+    application
+    idea
```

Comment:
:x: I would remove the `idea` plugin from the sample

Reply:
Will remove